Documentation/openshift.md

```diff
@@ -0,0 +1,107 @@
+---
+title: OpenShift
+weight: 17
+indent: true
+---
+
+# OpenShift
+
+[OpenShift](https://www.openshift.com/) adds a number of security and other enhancements to Kubernetes. In particular, [security context constraints](https://blog.openshift.com/understanding-service-accounts-sccs/) allow the cluster admin to define exactly which permissions are allowed to pods running in the cluster. You will need to define those permissions that allow the Rook pods to run. 
+
+## Rook Privileges
+
+To orchestrate the storage platform, Rook requires the following access in the cluster:
+- Create `hostPath` volumes, for persistence by the Ceph mon and osd pods
+- Run pods in `privileged` mode, for access to `/dev` and `hostPath` volumes
+- Host networking for the Rook agent and clusters that require host networking
```

Comment:
host networking is optional right? Might be good to specify which ones are optional.

Reply:
@bassam actually we need host networking in the agent. See my comment [here](https://github.com/rook/rook/pull/1729#discussion_r199242798) from a few days back. Jared and I did not find a way to remove the requirement on host network.